<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="large_gear_tray">
    <static>false</static>
    <allow_auto_disable>false</allow_auto_disable>

    <link name="large_gear_tray">
      <inertial auto="true"/>
      
      <visual name="large_gear_tray">
        <geometry>
          <mesh>
            <uri>model://demo_parts/large_gear_tray/meshes/large_gear_part_tray.stl</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>1.0 1.0 1.0 1.0</ambient>
          <diffuse>1.0 1.0 1.0 1.0</diffuse>
        </material>
      </visual>

      <collision name="large_gear_tray_collision">
        <density>20000</density>
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>model://demo_parts/large_gear_tray/meshes/large_gear_part_tray.stl</uri>
          </mesh>
          <!-- <box>
            <size> 0.210 0.138 0.015 </size>
          </box> -->
        </geometry>
        <max_contacts>5</max_contacts>
      </collision>
    </link>
  </model>
</sdf>